<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connect4frame</class>
 <widget class="QFrame" name="connect4frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,1,1,1,1" columnstretch="1,1,1,1,1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="3">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_16">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_30">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_10">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_40">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_19">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_18">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QLabel" name="label_28">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_39">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QLabel" name="label_41">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_36">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_33">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_14">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_12">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_9">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_29">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QLabel" name="label_32">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_20">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="label_34">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_23">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="label_38">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_27">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_22">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_31">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_37">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_24">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="label_21">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="6">
      <widget class="QLabel" name="label_35">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QLabel" name="label_26">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_25">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QLabel" name="label_42">
       <property name="styleSheet">
        <string notr="true">image: url(:/images/c4/empty.svg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
